### PR TITLE
Complete Sponsor packages integration. See #96.

### PR DESCRIPTION
**Pull request by ChrisHng**

### CODE REVIEW

File: docroot/themes/custom/dct_theme/js/matchHeight.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * @file
+ * DrupalCampTransylvania matchHeight behaviors.
+ */
+
+(function ($, Drupal) {
+
+  'use strict';
+
+  /**
+   *
```
Comment (lauracernau) [inline docroot/themes/custom/dct_theme/js/matchHeight.js]: Please add comment for what this functions does

File: docroot/modules/custom/dct_paragraphs/src/Plugin/ViewBuilder/SponsorshipContainerViewBuilder.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    // The box_container paragraph entity is retrieved.
+    $paragraph = $this->getEntity();
+
+    // The box list on the entity is retrieved.
+    $entities = $paragraph->get('field_sponsorship_item')->getValue();
+
+    foreach ($entities as $key => $value) {
+      $output[$row][] = $boxes[$key];
+
```
Comment (lauracernau) [inline docroot/modules/custom/dct_paragraphs/src/Plugin/ViewBuilder/SponsorshipContainerViewBuilder.php]: Please add a comment.

Comment (corneaalina) [inline docroot/modules/custom/dct_paragraphs/src/Plugin/ViewBuilder/SponsorshipContainerViewBuilder.php]: Pleas add a comment.

File: docroot/modules/custom/dct_paragraphs/src/Plugin/ViewBuilder/SponsorshipContainerViewBuilder.php [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+ * )
+ */
+class SponsorshipContainerViewBuilder extends EntityViewBuilderBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build(array &$build) {
+    $rows = $this->getBoxesInRows($build['field_sponsorship_item']);
+
```
Comment (corneaalina) [inline docroot/modules/custom/dct_paragraphs/src/Plugin/ViewBuilder/SponsorshipContainerViewBuilder.php]: Please add a comment.

File: docroot/modules/custom/dct_paragraphs/dct_paragraphs.info.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+name: DCT Map
+type: module
+description: Provides DCT flight connections map
```
Comment (danielpopdan) [inline docroot/modules/custom/dct_paragraphs/dct_paragraphs.info.yml]: Please correct the description.

File: docroot/modules/custom/dct_paragraphs/src/Plugin/ViewBuilder/SponsorshipContainerViewBuilder.php [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    // The box_container paragraph entity is retrieved.
+    $paragraph = $this->getEntity();
+
+    // The box list on the entity is retrieved.
+    $entities = $paragraph->get('field_sponsorship_item')->getValue();
+
+    foreach ($entities as $key => $value) {
+      $output[$row][] = $boxes[$key];
+
+      if (($key + 1) % 2 == 0) {
```
Comment (danielpopdan) [inline docroot/modules/custom/dct_paragraphs/src/Plugin/ViewBuilder/SponsorshipContainerViewBuilder.php]: Too complicated.. if( $key%2 != 0)

File: docroot/modules/custom/dct_paragraphs/src/Plugin/ViewBuilder/SponsorshipPackageViewBuilder.php [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  /**
+   * Based on the title of the paragraph returns the proper css class.
+   *
+   * @return mixed|null
+   *   Returns the needed class or null if the package type is not in $options.
+   */
+  protected function getClassFromPackageTitle() {
+    $title = $this->entity->get('field_title')->first()->value;
+    $options = [
+      'diamond',
```
Comment (danielpopdan) [inline docroot/modules/custom/dct_paragraphs/src/Plugin/ViewBuilder/SponsorshipPackageViewBuilder.php]: Is this needed? I think you can sanitize the title and set the class.

**Pull request merged**